docs/repo-docs/crafting-your-repository/caching.mdx

```diff
@@ -121,6 +121,8 @@ For information on how to connect your CI machines to Remote Cache, visit [the C
 
 ## What gets cached?
 
+Turborepo caches two types of outputs: Task outputs and Logs.
```

Comment:
I think that saying upfront there are two kinds of outputs helps linking them together. Both sections (Task outputs and Logs) are at the same level. In my mind, they were disconnected. My mental flow was:

from Task outputs
> If you do not declare file outputs for a task, Turborepo will not cache them.

So I thought _cache is disabled_. Then I read Logs, but I putted it in the context of _when cache is enabled_.

To better understand my confusion, you have to take into account that I was reading [Crafting your repository](https://turbo.build/repo/docs/crafting-your-repository) from top to bottom (the whole section). So, I read [Configuring tasks](https://turbo.build/repo/docs/crafting-your-repository/configuring-tasks#specifying-outputs) first.

In the `Specifying outputs` passage, there is no mention of "Logs". This lead me to think caching is disabled when `outputs` key is missing.

Later, by experimenting with Turborepo, I did not understand why my Lint task benefits from caching since I did not put `outputs` (because there is none from linters). And having Logs in the cache is very great for that matter!!!

Reply:
This is a good addition. It also seems like the suggestion from `docs/repo-docs/crafting-your-repository/configuring-tasks.mdx#218` would work better in the `Task Outputs` section below. What do you think?

Reply:
I'll do what you suggest. Now that I understand, I can't unlearn and figure out what would have been my train of thoughts on that. Let's wait the next newbie :)